api/handlers.go

```diff
@@ -2270,8 +2268,12 @@ func appendMediaLanguageFilterMods(exec boil.Executor, mods *[]qm.QueryMod, f Me
 		return nil
 	}
 
-	*mods = append(*mods, qm.Where("properties->>'original_language' = ?", f.MediaLanguage))
-
+	*mods = append(*mods,
+		qm.WhereIn("(id in ( SELECT DISTINCT cu.id FROM content_units cu "+
```

Comment:
please use ` for multiline strings in go instead of +

Reply:
Please add a TODO comment saying this query should be optimized ASAP and before we do that clients should use it as little as possible.

Reply:
[commit](https://github.com/Bnei-Baruch/archive-backend/pull/160/commits/671c9c5a75f9e22c75361769d43c7c71f89bf6df)